content/developer/reference/user_interface/view_architectures.rst

```diff
@@ -1702,17 +1734,12 @@ The `create` element can have the following attributes:
                  <field name="currency"/>
                  <field name="tax_id"/>
                  <control>
-                     <create string="Add a item"/>
+                     <create string="Add an item"/>
                      <create string="Add a section" context="{'default_type': 'section'}"/>
                      <create string="Add a note" context="{'default_type': 'note'}"/>
                  </control>
              </list>
 
-.. note::
-   Using the `control` element makes sense only if the list view is inside a
-   :class:`~odoo.fields.One2many` or :class:`~odoo.fields.Many2many` field. If any `create` element
-   is defined, it overwrites the default :guilabel:`add a line` button.
```

Comment:
Is this still the case? If so, it would be worth mentioning it in the `important` above.